lib/rubocop/cop/salsify/rails_application_mailer.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Salsify
+      # Check that mailers subclass ApplicationMailer with Rails 5.0
```

Comment:
I think the pattern existed, but it only started being auto-generated in rails 5.